pkg/services/featuremgmt/codeowners.go

```diff
@@ -9,18 +9,15 @@ const (
 	grafanaDashboardsSquad                      codeowner = "@grafana/dashboards-squad"
 	grafanaDatavizSquad                         codeowner = "@grafana/dataviz-squad"
 	grafanaFrontendPlatformSquad                codeowner = "@grafana/grafana-frontend-platform"
-	grafanaBackendGroup                         codeowner = "@grafana/grafana-backend-group"
```

Comment:
still needed (see comments in other file)

Reply:
readded